python/lsst/afw/math/detail/_convolve.cc

```diff
@@ -0,0 +1,141 @@
+                        "kernelPtr"_a, "bbox"_a, "xy0"_a, "doNormalize"_a, "bottomLeftImagePtr"_a,
+                        "bottomRightImagePtr"_a, "topLeftImagePtr"_a, "topRightImagePtr"_a);
+
+                clsKernelImagesForRegion.def("getBBox", &KernelImagesForRegion::getBBox);
+                clsKernelImagesForRegion.def("getXY0", &KernelImagesForRegion::getXY0);
+                clsKernelImagesForRegion.def("getDoNormalize", &KernelImagesForRegion::getDoNormalize);
+                clsKernelImagesForRegion.def("getImage", &KernelImagesForRegion::getImage);
+                clsKernelImagesForRegion.def("getKernel", &KernelImagesForRegion::getKernel);
+                clsKernelImagesForRegion.def("getPixelIndex", &KernelImagesForRegion::getPixelIndex);
+                clsKernelImagesForRegion.def("computeNextRow", &KernelImagesForRegion::computeNextRow);
+                clsKernelImagesForRegion.def_static("getMinInterpolationSize",
+                                                    KernelImagesForRegion::getMinInterpolationSize);
+            });
+
+    wrappers.wrapType(  py::enum_<KernelImagesForRegion::Location>(clsKernelImagesForRegion,"Location"),
+            [](auto &mod,auto &enm) {
+                enm.value("BOTTOM_LEFT", KernelImagesForRegion::Location::BOTTOM_LEFT);
+                enm.value("BOTTOM_RIGHT", KernelImagesForRegion::Location::BOTTOM_RIGHT);
+                enm.value("TOP_LEFT", KernelImagesForRegion::Location::TOP_LEFT);
+                enm.value("TOP_RIGHT", KernelImagesForRegion::Location::TOP_RIGHT);
+                enm.export_values();
+            });
+
+    wrappers.wrapType( py::class_<RowOfKernelImagesForRegion,
+                        std::shared_ptr<RowOfKernelImagesForRegion>>(wrappers.module,"RowOfKernelImagesForRegion"),
+            [](auto &mod,auto &clsRowOfKernelImagesForRegion) {
+                clsRowOfKernelImagesForRegion.def(py::init<int, int>(), "nx"_a, "ny"_a);
+
+                clsRowOfKernelImagesForRegion.def("front", &RowOfKernelImagesForRegion::front);
+                clsRowOfKernelImagesForRegion.def("back", &RowOfKernelImagesForRegion::back);
+                clsRowOfKernelImagesForRegion.def("getNX", &RowOfKernelImagesForRegion::getNX);
+                clsRowOfKernelImagesForRegion.def("getNY", &RowOfKernelImagesForRegion::getNY);
+                clsRowOfKernelImagesForRegion.def("getYInd", &RowOfKernelImagesForRegion::getYInd);
+                clsRowOfKernelImagesForRegion.def("getRegion", &RowOfKernelImagesForRegion::getRegion);
+                clsRowOfKernelImagesForRegion.def("hasData", &RowOfKernelImagesForRegion::hasData);
+                clsRowOfKernelImagesForRegion.def("isLastRow", &RowOfKernelImagesForRegion::isLastRow);
+                clsRowOfKernelImagesForRegion.def("incrYInd", &RowOfKernelImagesForRegion::incrYInd);
+            });
+}
+void wrapConvolve(lsst::utils::python::WrapperCollection &wrappers){
+     declareConvolve(wrappers);
```

Comment:
Needs several `addSignatureDependency` to support `Image` (probably won't work until you convert `afw.image`), as well as `Kernel`.